plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -29,6 +29,7 @@
 
 import java.net.URI;
 import java.util.List;
+import java.util.Optional;
```

Comment:
The commit body is too long. Please wrap at the end of the first sentence. 
https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

Reply:
Set up checks for this, hopefully you won't see this problem in my commits anymore!